frontend/apps/marketing/src/components/carousels/videoCarousel/VideoCarousel.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import '@code-dot-org/component-library/carousel/index.css';
+import DSCOCarousel, {
+  CarouselProps,
+} from '@code-dot-org/component-library/carousel';
+import React, {ReactNode} from 'react';
+
+import Video from '@code-dot-org/component-library/video';
+
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+interface VideoCarouselProps extends CarouselProps {
+  slides: {
+    id: string;
+    slide: ReactNode;
+    fields: {
+      videoTitle: string;
+      youTubeId: string;
+      videoFallbackFile: {fields: {file: {url: string}}};
```

Comment:
nit, non blocking: is there anything else useful in these objects that might be helpful to add in this types?

Reply:
I don't think so, these are the fields coming from the Video [content type](https://app.contentful.com/spaces/90t6bu6vlf76/environments/master/content_types/video/fields) in Contentful.